### ISSUE TITLE
Weather-Forecast overusing "Sun with Cloud" icon

### ISSUE THREAD

**Issue by AllesMeins** [pacjo/SmartspacerPlugins#12]
_Tl/dr: Weatherforecast seems to show the same icon for s wide variety of weather situations while other apps display a more nuanced selection of icons_

I'm not sure whether this is a bug or an artistic choice but to me it seems like the weather forecast plug-in is overusing the "Sun with Cloud" icon a bit. I've been using the plug-in for a few months now and it seems like 90% of the time I only see this one icon. 

At the first few days I was certain that the plug-in was broken because it always showed the same icon regardless of the weather condition outside ranging from "almost completely sunny" to "completely overcast".

So my question would be: Is this intentional? 
Take for example those screenshots I took this morning.

**What the plug-in showed:**
![Screenshot_20240722-074500](https://github.com/user-attachments/assets/10429f43-8e63-4076-a6ec-f58499f0f6d3)

**What Breezy (which I use as the weather backend) showed for the same time:** 
![Screenshot_20240722-074543](https://github.com/user-attachments/assets/0d9fe715-9139-4926-a691-d122dc2f4207)

**What my window showed 😁**
![PXL_20240722_054720842](https://github.com/user-attachments/assets/838ad30e-7543-4169-aa4c-7635706f4369)

**Comment by pacjo** [comment]
I think that's fixed on git, but not released. Though I'm not sure. My projects are a mess and constantly starting new ones doesn't help.

In any case, thanks for the info. I'll see if it's possible to make an update in the current state.

**Comment by pacjo** [comment]
Alright, there weren't that many changes. So there's a new release. Let me know if it fixes the issue.

Though looking at breezy forecast icons will probably be mostly the same. The mapping I use needs to be fixed, but no matter what I choose it always looks bad.

**Comment by pacjo** [comment]
@AllesMeins did [v1.4](https://github.com/pacjo/SmartspacerPlugins/releases/tag/v1.4) fixed the issue?

**Comment by AllesMeins** [comment]
Thank you for your quick reaction. I've installed the newest version and watching the issue. "Unfortunately" the weather was sunny here today, so I can't say whether the cloudy-conditions are displayed properly at the moment.

**Comment by AllesMeins** [comment]
Hi, checked it today - unfortunately it didn't change. This morning I had the same situation - breezy showing all cloudy and gray while the plug-in showed only the sun with clouds again.

**Comment by pacjo** [comment]
Can you share the cloud cover from breezy?

It's working as expected on mine (though maybe we can have smaller sun with ~90% cloud cover, I'll see what is available)

![Screenshot_20240726-130925_Lawnchair (Debug)~2](https://github.com/user-attachments/assets/1f177b38-cdc3-4a69-acef-76f546024b3d)
![Screenshot_20240726-131009_Breezy Weather](https://github.com/user-attachments/assets/8d1831b9-d642-445f-b306-ea9b7cb585af)

**Comment by AllesMeins** [comment]
Ah, I see. Breezy currently reports a cloud cover of about 87-89% which is already displayed with a completely cloudy icons. This seems fitting because in real live those 87% translate to gray sky with patches of lighter gray - at least where I'm living.

![Screenshot_20240722-074543_1](https://github.com/user-attachments/assets/84bfa8a0-ee18-4a7f-aa53-1c57a1f57527)
![Screenshot_20240726-160354_1](https://github.com/user-attachments/assets/864914b8-aa25-4a92-a246-92b1437fb3fb)

 So I would go for an icon with maybe multiple clouds and only a tiny bit of sun. Something along the lines of this: https://cdn2.iconfinder.com/data/icons/weather-filled-outline-3/64/weather03-1024.png (of cause only as the general idea and not in this style :-) )

**Comment by pacjo** [comment]
Ok, I'll try to tweak the mapping, but I just noticed a funny thing.

Turns out Google too has a rather optimistic weather.

(top - mine; bottom - google; right - breezy)
![Screenshot_20240802-151006_Pixel Launcher](https://github.com/user-attachments/assets/68ebdde7-3f66-48d3-8159-af3bd88cbbfd)
